server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultServiceContext.java

```diff
@@ -680,83 +686,87 @@ private void injectServiceOptions(String transportURI, Map<String, Object> optio
-                    options.put(format("http[http/1.1].%s", REALMS),
-                    		realms);
-                    for (String optionPattern : asList("http[httpxe/1.1].%s", "http[x-kaazing-handshake].%s")) {
-                        options.put(format(optionPattern, REQUIRED_ROLES),
-                                getRequireRoles());
-                        options.put(format(optionPattern, REALMS),
-                        		realms);
-                        // We need this to support reading legacy service properties during authentication.
-                        // authentication-connect, authentication-identifier, encryption.key.alias, service.domain
-                        // The negotiate properties are replaced with client-side capabilities to use different
-                        // KDCs on a per-realm basis. The ksessionid cookie properties are needed to write the cookie
-                        // out if needed (recycle authorization mode).
-                        options.put(format(optionPattern, AUTHENTICATION_CONNECT),
-                                getProperties().get("authentication.connect"));
-                        options.put(format(optionPattern, AUTHENTICATION_IDENTIFIER),
-                                getProperties().get("authentication.identifier"));
-                        options.put(format(optionPattern, ENCRYPTION_KEY_ALIAS),
-                                getProperties().get("encryption.key.alias"));
-                        options.put(format(optionPattern, SERVICE_DOMAIN),
-                                getProperties().get("service.domain"));
-                    }
-                }
-
-                // TCP
-                for (String optionPattern : asList("tcp.%s")) {
-                    // NO REALM_NAME as this will be an accept/connect option
-                    String tcpRealmOptionName = format(optionPattern, "realm");
-                    String tcpRealmName = (String) options.get(tcpRealmOptionName);
-                    if (tcpRealmName != null) {
-                        // TODO, use REALMS TCP connect options
-                        // check if it's the same as the configured realm
-                        if (!serviceRealmContext.getName().equals(tcpRealmName)) {
-                            logger.error("{} configuration error: {} needs to be set to the same value as the configured realm",
-                                    serviceName, tcpRealmOptionName);
-                            throw new IllegalArgumentException(
-                                    tcpRealmOptionName + " needs to be the same as the configured realm");
-                        }
-                        options.put(format(optionPattern, "loginContextFactory"), serviceRealmContext.getLoginContextFactory());
-                    }
-                }
+        final String httpRealmOptions = (String) options.remove("http.realm");
+
+//        TODO DPW
+//        if (serviceRealmContext == null &&
```

Comment:
This is a check we had, but there is no matching tests.  I think I will remove it